conferences/views.py

```diff
@@ -51,12 +54,22 @@ def user_preferences_edit(request, user_preferences_id=None):
 
 @staff_member_required()
 @require_http_methods(['POST'])
-def toggle_payment_accepted(request):
+def admin_edit(request):
     user_preferences_id = request.POST.get('key', None)
     user_preferences = get_object_or_404(UserPreferences, pk=user_preferences_id)
-    user_preferences.toggle_payment_accepted()
-    user_preferences.save()
-    return JsonResponse({'msg': "{} changed status!".format(user_preferences.user.get_full_name())})
+    command = request.POST.get('command', False)
+    if command == 'toggle_payment_accepted':
```

Comment:
I think it might be nice to keep command name in some variable and pass it here and to template. It would be easier to keep it safe from bugs and bad luck